server/src/service/item.rs

```diff
@@ -0,0 +1,57 @@
+// org.freedesktop.Secret.Item
+
+use std::collections::HashMap;
+
+use oo7::dbus::api::properties::Properties;
+use oo7::dbus::api::Secret;
+use serde::{Deserialize, Serialize};
+use zbus::{dbus_interface, fdo, zvariant, ObjectServer, SignalContext};
+use zvariant::{ObjectPath, OwnedObjectPath};
+
+#[derive(Debug, Serialize, Deserialize)]
+pub struct Item {
+    locked: bool,
+    attributes: Properties,
+    label: &'static str,
```

Comment:
The properties already contain the label, so you can just store that

Reply:
done, thanks!